crates/plume/src/expand_message_xmd.nr

```diff
@@ -52,7 +53,7 @@ fn msg_prime<let N: u32>(msg: [u8; N]) -> [u8; 32] {
         preimage[64 + N + 2 + 1 + i] = DST_PRIME[i];
     }
 
-    sha256(preimage)
+    hash([to_field(preimage)], preimage.len()).to_le_bytes()
```

Comment:
Since this is not quite part of the `PLUME`, but rather of the hash-to-curve algorithm, it might be not conforming to requirements given in [spec](https://www.ietf.org/archive/id/draft-irtf-cfrg-hash-to-curve-13.html#name-expand_message_xmd). Though, I am not particularly sure

Reply:
I think if I hash the whole thing by breaking it into limbs and hashing the limbs, it would conform to the spec. But I am not a cryptographer, so I may be wrong